pkg/skaffold/schema/v1beta7/upgrade.go

```diff
@@ -26,7 +26,7 @@ import (
 // Upgrade upgrades a configuration to the next version.
 // Config changes from v1beta7 to v1beta8
 // 1. Additions:
-// kaniko/resource requirements
+//    gitTagger/variant
```

Comment:
actually, I think this now needs to go in `pkg/skaffold/schema/v1beta8/upgrade.go`

Reply:
@nkubala You have keen eyes!